.github/workflows/convert-to-pdf.yml

```diff
@@ -0,0 +1,31 @@
+name: Wersje tekstowe to PDF
+# This workflow is triggered on pushes to the repository.
+on:
+  push:
+    branches:
+      - main
+    # Paths can be used to only trigger actions when you have edited certain files, such as a file within the /docs directory
+    paths:
+      - 'wersje tekstowe/**.md'
+      - 'wersje tekstowe/images/**'
+
+jobs:
+  converttopdf:
+    name: Build PDF
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: baileyjm02/markdown-to-pdf@v1.2.0
+        with:
+          input_dir: wersje tekstowe
+          output_dir: wersje pdf
+          images_dir: wersje tekstowe/images
```

Comment:
Maybe stick to Polish and change `images` to `obrazy`? It would be nice to create an empty folder and place `.gitkeep` inside until the 1st image is created

Reply:
I would also consider removing empty spaces from file and directory names.
It may cauze unexpected results while used with software not supporting it.